Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5.4-apache
+RUN apt-get update
+RUN apt-get install -y zip libxml2-dev libpng-dev
+RUN docker-php-ext-install mysql simplexml mbstring gd mysqli
+RUN pecl install xdebug-2.4.0 && docker-php-ext-enable xdebug
+RUN echo "xdebug.remote_enable=on" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
+RUN echo "xdebug.remote_connect_back=on" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
+# nasty hack to solve perms issues
+RUN chmod 777 /var/www/html
```

Comment:
This should be more specific to only directories that need to be writable (e.g. attachments)

Reply:
Absolutely, I agree.  Docker didn't like it when I tried to do it that way.  Before this gets merged (if accepted), I will fix this. I just need to find the solution.

My first priority is to get someone else to test this.

I work on Linux & mac. Going to try my macbook tomorrow.

Reply:
Also, `chmod 775` should be sufficient.